module-code/app/securesocial/controllers/LoginApi.scala

```diff
@@ -32,13 +38,21 @@ import securesocial.core.services.SaveMode
 /**
  * A default controller that uses the BasicProfile as the application user type.
  */
-class LoginApi @Inject() (implicit val env: RuntimeEnvironment, val configuration: Configuration, val playEnv: Environment) extends BaseLoginApi
+class LoginApi @Inject() (implicit val env: RuntimeEnvironment, val configuration: Configuration, val playEnv: Environment,
+    langs: Langs,
+    action: DefaultActionBuilder,
+    parsers: PlayBodyParsers,
+    implicit val messagesApi: MessagesApi,
```

Comment:
You don't need the `implicit` here and the subsequent ones.
See the page below:
https://stackoverflow.com/questions/10635406/defining-a-function-with-multiple-implicit-arguments-in-scala